app/assets/javascripts/discourse/tests/integration/components/admin-schema-theme-setting/editor-test.gjs

```diff
@@ -1560,3 +1563,502 @@ module(
     });
   }
 );
+
+module(
```

Comment:
Instead of passing a theme setting or plugin setting object through, I think we should serialize those settings into a standard set of arguments for the component. The editor doesn't need to know about the type of object.